chart/k8gb/values.yaml

```
@@ -35,6 +34,7 @@ externaldns:
     runAsNonRoot: true
 
 coredns:
+  # service: refer to docs/service_upgrade.md for upgrading CoreDNS service steps
```

this part can be viewed in a standalone chart context where the relative `docs` path does not make sense.
```suggestion
  # service: refer to https://www.k8gb.io/docs/service_upgrade.html for upgrading CoreDNS service steps
```

---

@k0da how is it resolved?

---

@ytsarev sorry I missed it. Will amend

---

@ytsarev done